[DOCU-2759] Updated Service Hub Overview & Service Overview (#5010)
Kong/docs.konghq.com: app/konnect/servicehub/index.md

---
title: The Konnect Service Hub
subtitle: Track every service accross your architecture
content-type: explanation
---

The Service Hub makes internal APIs discoverable, consumable, and reusable for internal development teams. Catalog all your services through the Service Hub to create a single source of truth for your organization’s service inventory. By leveraging Service Hub, your application developers can search, discover, and consume existing services to accelerate their time-to-market, while enabling a more consistent end-user experience across the organization’s applications.

**[Access the {{site.konnect_short_name}} service hub](https://cloud.konghq.com/us/servicehub).**

![{{site.konnect_short_name}} service hub](/assets/images/docs/konnect/konnect-servicehub.png)

> ## Konnect Service Dashboard

Reviewer: ```suggestion
## Service Package Dashboard
```

Reviewer: incorporating feedback from Sam

> The Konnect Service Package Dashboard is the place to manage services, versions, and documentation. The dashboard is available by clicking any Service Package in the Service Hub. 

Reviewer: ```suggestion
The Service Package Dashboard is the place to manage services, versions, and documentation. The dashboard is available by clicking any Service Package in the Service Hub. 
```

Here are some of the things you can do from the Service Package Dashboard: 

* Configure a service
* Publish a service to the Dev Portal
* Manage versions and labels. 
* View traffic, error, and latency data. 

![{{site.konnect_short_name}} service hub](/assets/images/docs/konnect/konnect-service-package-versions.png)

Number | Item | Description
-------|------|------------

> 1 | **Service Versions** | This section displays the status of a service version. From the context menu you can **Delete** a service, or use the **View Details** button to navigate to that versions dashboard. 

Reviewer: ```suggestion
1 | **Service Versions** | This section displays the status of a service version. From the context menu you can **Delete** a service version, or use the **View Details** button to navigate to that versions dashboard. 
```

2 | **Analytics** | Analytics data for the service. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/)

> 3 | **Documentation** | You can add markdown documentation for your service. The documentation will render in the dashboard. If the service is published, the documentation will be available to developers in the Dev Portal. 

Reviewer: ```suggestion
3 | **Documentation** | You can add markdown documentation for your service, as well as an API specification for each version of the service. You can control the individual publishing status of each document you upload to a service.
```

For example, if you have three runtime groups for `development`, `staging`, and
`production`, you can manage which environment the service is available in by
assigning a version to that group at creation time. You might have v1 running
in `production`, and be actively working on v2 in `development`. Once it's
ready to test, you'd create v2 in `staging` before finally creating v2 in
`production` alongside v1.

### Analytics

The analytics dashboard shown in the **Service Dashboard** is a high level overview of **traffic**, **error**, and **latency** for the service. These reports are generated automatically based on the traffic to the service. 

Learn more: 

* [Analytics overview](/konnect/analytics/)
* [How to analyze services and routes](/konnect/analytics/services-and-routes/)
* [How to generate reports](/konnect/analytics/generate-reports/)

### Documentation

The **Service Dashboard** can be used to manage documentation for your service. Documentation can be either an API spec, or Markdown documentation for the service. Once the documentation is uploaded it can be edited from the dashboard. The documentation can be accessed once the service is published.

Learn more: 

> * [Manage service documentation](/konnect/servicehub/service-doumentation/)

Reviewer 2: ```suggestion
* [Manage service documentation](/konnect/servicehub/service-documentation/)
```

---
title: The Konnect Service Hub
subtitle: Track every service accross your architecture
content-type: explanation
---

The Service Hub makes internal APIs discoverable, consumable, and reusable for internal development teams. Catalog all your services through the Service Hub to create a single source of truth for your organization’s service inventory. By leveraging Service Hub, your application developers can search, discover, and consume existing services to accelerate their time-to-market, while enabling a more consistent end-user experience across the organization’s applications.

![{{site.konnect_short_name}} service hub](/assets/images/docs/konnect/konnect-servicehub.png)

> **[Access the {{site.konnect_short_name}} service hub](https://cloud.konghq.com/us/servicehub).**

Reviewer 3: I thought this was a little buried, maybe put this before the screenshot with one of those arrows showing it's an external link?
